### ISSUE TITLE
Popover is not trapping focus

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#1266]
Popover should be moving focus and trapping it from tabbing out of Popover.

http://design-system-react-components.herokuapp.com/?selectedKind=SLDSPopover&selectedStory=Header&full=0&addons=1&stories=1&panelRight=0
